enum.go

```diff
@@ -112,10 +112,33 @@ func AllConnectiveEnum() []string {
 	}
 }
 
+// ContactType represents the method of contact.
+type ServiceOwnershipCheckContactType string
+
+const (
+	ServiceOwnershipCheckContactTypeAny         ServiceOwnershipCheckContactType = "any"          // A Slack channel contact method.
+	ServiceOwnershipCheckContactTypeSlack       ServiceOwnershipCheckContactType = "slack"        // A Slack channel contact method.
+	ServiceOwnershipCheckContactTypeSlackHandle ServiceOwnershipCheckContactType = "slack_handle" // A Slack handle contact method.
+	ServiceOwnershipCheckContactTypeEmail       ServiceOwnershipCheckContactType = "email"        // An email contact method.
+	ServiceOwnershipCheckContactTypeWeb         ServiceOwnershipCheckContactType = "web"          // A website contact method.
+)
+
+// All ContactType as []string
+func AllServiceOwnershipCheckContactType() []string {
+	return []string{
+		string(ServiceOwnershipCheckContactTypeAny),
+		string(ServiceOwnershipCheckContactTypeSlack),
+		string(ServiceOwnershipCheckContactTypeSlackHandle),
+		string(ServiceOwnershipCheckContactTypeEmail),
+		string(ServiceOwnershipCheckContactTypeWeb),
+	}
+}
+
 // ContactType represents the method of contact.
 type ContactType string
 
 const (
+	ContactTypeAny         ContactType = "any"        // A Slack channel contact method.
```

Comment:
Should this not be here then if it's not a valid enum value on this contact type?

Reply:
ah thank you, I accidentally left that in

Reply:
removed in latest commit